debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -379,7 +379,7 @@ public static BinaryHandlingMode parse(String value, String defaultValue) {
             .withWidth(Width.SHORT)
             .withImportance(Importance.LOW)
             .withValidation(CommonConnectorConfig::validateSkippedOperation)
-            .withDescription("The comma-separated list of operations to skip during streaming, defined as: 'i' for inserts; 'u' for updates; 'd' for deletes. "
+            .withDescription("The comma-separated list of operations to skip during streaming, defined as: 'c' for inserts/create; 'u' for updates; 'd' for deletes. "
```

Comment:
That's not quite correct actually. There is "r" for read (snapshot), "c", "u", and "d".

Reply:
Is it possible to skip `r` events?